src/gui_common/dialogs/CustomDialog.cs

```diff
@@ -395,6 +442,38 @@ protected override void ApplyRectSettings()
         }
     }
 
+    protected void ConnectToWindowReorderingNodes()
+    {
+        var windowReorderingAncestorsIEnumerable = AddWindowReorderingSupportToSiblings.GetWindowReorderingAncestors(
+            this, AutomaticWindowReorderingDepth, WindowReorderingPaths);
+
+        foreach (var windowReordering in windowReorderingAncestorsIEnumerable)
```

Comment:
```suggestion
        foreach (var (reorderingNode, nodeSibling) in windowReorderingAncestorsIEnumerable)
```
I think destructuring the list of tuples should work like that...